build.py

```diff
@@ -2,10 +2,13 @@
 # -*- coding: utf-8 -*-
 
 
-from bincrafters import build_template_default
+from bincrafters import build_shared, build_template_default
 
 if __name__ == "__main__":
 
-    builder = build_template_default.get_builder()
+    builder = build_shared.get_builder()
```

Comment:
What are these changes about?

Reply:
This only provides a binary, no libraries.
So adding builds for all combinations of build_type and compiler does not make sense.

Reply:
For example: https://dl.bintray.com/bincrafters/public-conan/bincrafters/winflexbison/2.5.16/stable/package/3ca85915ce57a6607676849c47c2cee02c75209c/conanmanifest.txt

no `/lib`

Reply:
Tbh, this is surprising that this package generates no libs.  Indeed if it only produces binaries, `_installer` might make sense.  But, as @Croydon said, it's accepted already, so lets publish under current name for now and discuss rename later.

Reply:
The [install rules in the cmake file install the files in the root package folder](
https://github.com/lexxmark/winflexbison/blob/a72f3d6b5102b65f064a5054ba634d3d62e94f41/CMakeLists.txt#L37-L47).
You can either `tools.replace_in_file` all these lines, or simply copy these files manually after it installed itself. I choose the latter because it is easier.

Reply:
Note that using flex for on visual studio, requires the `--wincompat` option.